ci-operator/step-registry/ipi/conf/gcp/osdisk/disktype/compute/ipi-conf-gcp-osdisk-disktype-compute-commands.sh

```diff
@@ -73,7 +73,7 @@ else
   if version_check "${EXPECTED_OCP_VERSION}"; then
     valid_types=("pd-balanced" "pd-ssd" "pd-standard")
   else
-    valid_types=("pd-ssd" "pd-standard")
+    valid_types=("pd-ssd" "pd-standard" "hyperdisk-balanced")
```

Comment:
I do not think `hyperdisk-balanced` is supported by 4.14- version of OCP, if you need the disk type, you need to overwrite it by setting `COMPUTE_DISK_TYPE` explictly.